domain-rrd/rrd-workflows/src/main/java/gov/va/vro/service/provider/MasProcessEntry.java

```diff
@@ -0,0 +1,23 @@
+package gov.va.vro.service.provider;
+
+import gov.va.vro.model.rrd.mas.MasAutomatedClaimPayload;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class MasProcessEntry implements Processor {
+  private final CamelEntrance camelEntrance;
+
+  @Override
+  @SneakyThrows
+  public void process(Exchange exchange) {
+    var claimPayload = exchange.getMessage().getBody(MasAutomatedClaimPayload.class);
+    camelEntrance.processClaim(claimPayload);
```

Comment:
This is very strange. You have a processor that's....entering back into camel to process the body?